### ISSUE TITLE
Optimise Code

### ISSUE THREAD

**Issue by Anushika09** [Anushika09/SignUp#5]
I Want help in optimize current code

**Comment by ashikvarma11** [comment]
@Anushika09  Can I work on this? Please assign me the issue.

**Comment by iyosayi** [comment]
Can I work on this too? Please assign.

**Comment by iyosayi** [comment]
Thank you @Anushika09

**Comment by iyosayi** [comment]
Hey, @Anushika09 are we allowed to restructure the entire backend files and code? I want to use the `Clean code Architecture` to achieve this.

**Comment by Anushika09** [comment]
Yes @iyosayi  you can restructure the backend file but make sure it's clean

**Comment by iyosayi** [comment]
Okay, I am on it already. @Anushika09

**Comment by amitkv1212** [comment]
hey @anushika im interested in doping it could you please assign it to me
